docs/example.nginx.conf

```diff
@@ -34,15 +34,15 @@ server {
     # Will not set any header if it is emptystring
     add_header Cache-Control $cacheControl;
 
-    set $styleSrc   "'unsafe-inline' 'self' your-main-domain.com";
-    set $scriptSrc  "'self' your-main-domain.com";
-    set $connectSrc "'self' https://your-main-domain.com wss://your-main-domain.com https://api.your-main-domain.com wss://your-main-domain.com your-main-domain.com blob: your-main-domain.com";
-    set $fontSrc    "'self' data: your-main-domain.com";
-    set $imgSrc     "data: * blob:";
-    set $frameSrc   "'self' your-sandbox-domain.com blob:";
-    set $mediaSrc "* blob:";
-    set $childSrc "https://your-main-domain.com";
-    set $workerSrc "https://your-main-domain.com";
+    set $styleSrc	"'unsafe-inline' 'self' your-main-domain.com";
+	set $scriptSrc	"'self' your-main-domain.com";
+	set $connectSrc	"'self' https://your-main-domain.com wss://your-main-domain.com your-main-domain.com https://api.your-main-domain.com blob: your-main-domain.com";
+	set $fontSrc	"'self' data: your-main-domain.com";
+	set $imgSrc	    "'self' data: * blob: your-main-domain.com";
+	set $frameSrc	"'self' your-sandbox-domain.com blob: your-sandbox-domain.com";
+	set $mediaSrc	"'self' data: * blob: your-main-domain.com";
```

Comment:
the addition of  `data:` doesn't seem strictly necessary, but I don't think it opens up any risk of attack.